hooks/useIntlTelNumbers.js

```diff
@@ -0,0 +1,365 @@
+import { useState } from 'react';
+import 'libphonenumber-js-utils';
```

Comment:
This lib is an interesting one.
As said in the README.md of the package (<https://www.npmjs.com/package/libphonenumber-js-utils>), it is a compiled and minified version of the utilities functions from the googlei18n's libphonenumber library. The functions are exposed to window.intlTelInputUtils inside browser environment.

We may want to clone googles lib and use it w/o 3rd parties, but it looks quite complex and I didn't managed to wrap my head around how to use it in reasonable timeframe so I leave it open for a discussion.

Reply:
What do you think about using https://github.com/catamphetamine/libphonenumber-js instead? It seems to have a smaller footprint.